src/main/java/jenkins/branch/buildstrategies/basic/AllBranchBuildStrategyImpl.java

```diff
@@ -0,0 +1,143 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2018, CloudBees, Inc.
```

Comment:
do you work for cloudbees? MIT license needs to be attributed correctly. If you have copied and pasted significant work from CloudBees code then just append your name or organization, otherwise don't include the CloudBees and just put your name